packages/x-telemetry/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@mui/x-telemetry",
+  "version": "8.0.0-alpha.12",
+  "description": "MUI X Telemetry",
+  "author": "MUI Team",
+  "main": "src/index.ts",
+  "license": "SEE LICENSE IN LICENSE",
+  "bugs": {
+    "url": "https://github.com/mui/mui-x/issues"
+  },
+  "homepage": "https://mui.com/x/guides/telemetry/",
+  "publishConfig": {
+    "access": "public",
+    "directory": "build"
+  },
+  "scripts": {
+    "typescript": "tsc -p tsconfig.json",
+    "build": "pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files ",
+    "build:modern": "node ../../scripts/build.mjs modern",
+    "build:node": "node ../../scripts/build.mjs node",
+    "build:stable": "node ../../scripts/build.mjs stable",
+    "build:copy-files": "node ../../scripts/copyFiles.mjs && node ./scripts/addPackageScripts.js",
+    "build:types": "tsx ../../scripts/buildTypes.mts",
+    "prebuild": "rimraf build tsconfig.build.tsbuildinfo"
+  },
+  "sideEffects": false,
+  "packageScripts": {
+    "postinstall": "node ./postinstall/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-x.git",
+    "directory": "packages/x-telemetry"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.26.9",
+    "@fingerprintjs/fingerprintjs": "^4.6.0",
+    "@mui/utils": "^5.16.6 || ^6.0.0",
+    "ci-info": "^4.0.0",
+    "conf": "^5.0.0",
```

Comment:
We need to remove this dependency. Issue created https://github.com/mui/mui-private/issues/804.